_layouts/default.html

```diff
@@ -335,8 +335,6 @@
                                         <ul class="nav navbar-nav">
                                             <li {% if '/help/' == page.url %}class="active" {% endif %}><a href="{{ site.url }}/help/"><i class="fa fa-fw fa-users"></i> Community help</a></li>
                                             <li {% if '/bug-bounties/' == page.url %}class="active" {% endif %}><a href="{{ site.url }}/bug-bounties/"><i class="fa fa-fw fa-usd"></i> Bug bounties</a></li>
-                                            <li {% if '/consulting/' == page.url %}class="active" {% endif %}><a href="{{ site.url }}/consulting/"><i class="fa fa-fw fa-user-md"></i> Professional help</a></li>
```

Comment:
`consulting.md` and `training.md` should probably be removed, since nothing links to them anymore

Reply:
Indeed! I remember removing pages causes some warnings in Google Webmaster tools, but nevermind, it's just not good to have old orphaned pages

Reply:
Ok, I'll remove these 2 pages